.env.example

```
@@ -12,4 +12,6 @@ NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=your_API_key
 CLERK_SECRET_KEY=your_API_key
 
 # completely not obligatory
-OPENAI_API_KEY=your_OpenAI_key
\ No newline at end of file
+OPENAI_API_KEY=your_OpenAI_key
+
+INNGEST_BASE_URL="http://127.0.0.1:8288"
```

Could you try (and if it works, change it to):

```suggestion
INNGEST_DEV=1
```

I suspect it's a server actions bug with Next.js where it can’t detect the node env and it tries to use prod instead of dev server. I'm a bit surprised because I haven't encountered that when working on the boilerplate. Thank you!

More info on the env variable:
https://www.inngest.com/docs/sdk/environment-variables#inngest-dev

---

Thank you, the dev mode worked well.